src/style.css

```diff
@@ -0,0 +1,156 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card {
+  width: 200px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__apple-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: hsl(234, 80%, 12%);
+  margin: 0 16px;
+
+  width: 166px;
+  height: 36px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  width: 92px;
+  height: 14px;
+
+  color: #616070;
+  margin: 4px 16px 16px;
+
+  text-align: left;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 16px;
+  line-height: 14px;
+
+  margin-left: 16px;
+  margin-bottom: 8px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-image: url(images/star.svg);
+
+  display: flex;
+}
+
+.stars :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars :not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars {
+  display: flex;
+  align-items: flex-start;
+}
+
+.stars-text {
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  color: rgb(6, 11, 53);
+  font-weight: 400;
+  margin-right: 16px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  align-items: center;
+}
+
+.price__text {
+  line-height: 18px;
+  font-weight: 400;
+  font-size: 12px;
+  text-align: left;
+  color: #616070;
+  margin-left: 16px;
+}
+
+.price__apple {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: hsl(234, 80%, 12%);
+  margin-right: 16px;
+}
+
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 166px;
```

Comment:
it's redundnat, with display: flex your button will ttake up all of the available width